src/foam/u2/ActionView.js

```diff
@@ -141,7 +145,7 @@ foam.CLASS({
           this.onDetach(cRSlot$.sub(() => this.setConfirm(cRSlot$.get())));
           this.setConfirm(cRSlot$.get());
         }
-        this.attrs({name: this.action.name, 'aria-label': this.action.ariaLabel });
+        this.attrs({name: this.action.name, 'aria-label': this.ariaLabel });
```

Comment:
Isn't this already done in Button, so will already happen when this calls this.SUPER()?

Funny that it was trying to set the aria-label with this.ariaLabel, but that that property didn't exist. Seems like an incomplete port.

Reply:
Yup, I think i missed this when we moved to Button.js. For actions it would pick up the ariaLabel on the action definition but other button tags would get nothing. Either ways, fixed now.